rust-bitcoin-derive/src/lib.rs

```diff
@@ -0,0 +1,87 @@
+//! Custom derive macros for the rust-bitcoin project.
```

Comment:
I'd love to stop prefixing things with `rust-` outside of ambiguous communication. I believe people can remember the parent is written in Rust and expect that child is as well especially given the `-derive` suffix which is common in Rust libs.

Reply:
Good point.